+ source /root/mugen/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n '' ']'
+ trap POST_TEST_DEFAULT EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n '' ']'
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start executing testcase.'
+ message='Start executing testcase.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start executing testcase.'
Sat Apr 29 02:44:41 2023 - INFO  - Start executing testcase.
+ find /etc/ssh/sshd_config -type f -user root -group root -perm 600
/etc/ssh/sshd_config
+ CHECK_RESULT 0 0 0 'check /etc/ssh/sshd_config file right fail'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='check /etc/ssh/sshd_config file right fail'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ find /etc/sysctl.conf -type f -user root -group root -perm 600
+ CHECK_RESULT 0 0 0 'check /etc/sysctl.conf file right fail'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='check /etc/sysctl.conf file right fail'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ find /lib/ -name modules -type d -user root -group root -perm 750
+ CHECK_RESULT 0 0 0 'check /lib/modules/ file right fail'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='check /lib/modules/ file right fail'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ find /root/ -type f -user root -group root -perm 700
+ CHECK_RESULT 0 0 0 'check /root/ file right fail'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='check /root/ file right fail'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ find /tmp/ -type f -user root -group root -perm 1777
+ CHECK_RESULT 0 0 0 'check /tmp/ file right fail'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='check /tmp/ file right fail'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ find /dev/shm -type f -user root -group root -perm 1777
+ CHECK_RESULT 0 0 0 'check /dev/shm file right fail'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='check /dev/shm file right fail'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ find /var/log/ -name audit -type d -user root -group root -perm 750
+ CHECK_RESULT 0 0 0 'check /var/log/audit/ file right fail'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='check /var/log/audit/ file right fail'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ find /var/log/audit/audit.log -type f -user root -group root -perm 600
find: ‘/var/log/audit/audit.log’: No such file or directory
+ CHECK_RESULT 1 0 0 'check /var/log/audit/audit.log file right fail'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='check /var/log/audit/audit.log file right fail'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'check /var/log/audit/audit.log file right fail'
+ LOG_ERROR 'check /var/log/audit/audit.log file right fail'
+ message='check /var/log/audit/audit.log file right fail'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'check /var/log/audit/audit.log file right fail'
Sat Apr 29 02:44:48 2023 - ERROR - check /var/log/audit/audit.log file right fail
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_check_file_sys_protect_003.sh line 48'
+ message='oe_test_check_file_sys_protect_003.sh line 48'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_check_file_sys_protect_003.sh line 48'
Sat Apr 29 02:44:49 2023 - ERROR - oe_test_check_file_sys_protect_003.sh line 48
+ '[' 0 -eq 1 ']'
+ return 0
+ find /var/ -name log -type d -user root -group root -perm 750
+ CHECK_RESULT 0 0 0 'check /var/log/ file right fail'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='check /var/log/ file right fail'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
++ 'find /var/log/* -type f -user root -group root -perm 640 | wc -l'
oe_test_check_file_sys_protect_003.sh: line 53: find /var/log/* -type f -user root -group root -perm 640 | wc -l: No such file or directory
+ getFileNum=
++ 'find /var/log/* -type f | wc -l'
oe_test_check_file_sys_protect_003.sh: line 54: find /var/log/* -type f | wc -l: No such file or directory
+ allFileNum=
+ test '' -eq ''
oe_test_check_file_sys_protect_003.sh: line 55: test: : integer expression expected
+ CHECK_RESULT 2 0 1 'check /var/log/* file right fail'
+ actual_result=2
+ expect_result=0
+ mode=1
+ error_log='check /var/log/* file right fail'
+ exit_mode=0
+ '[' -z 2 ']'
+ '[' 1 -eq 0 ']'
+ test 2x == 0x
+ return 0
+ '[' -e /var/log/secure ']'
+ find /var/log/auth.log -type f -user root -group root -perm 640
find: ‘/var/log/auth.log’: No such file or directory
+ CHECK_RESULT 1 0 0 'check /var/log/secure file right fail'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='check /var/log/secure file right fail'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'check /var/log/secure file right fail'
+ LOG_ERROR 'check /var/log/secure file right fail'
+ message='check /var/log/secure file right fail'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'check /var/log/secure file right fail'
Sat Apr 29 02:44:50 2023 - ERROR - check /var/log/secure file right fail
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_check_file_sys_protect_003.sh line 63'
+ message='oe_test_check_file_sys_protect_003.sh line 63'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_check_file_sys_protect_003.sh line 63'
Sat Apr 29 02:44:51 2023 - ERROR - oe_test_check_file_sys_protect_003.sh line 63
+ '[' 0 -eq 1 ']'
+ return 0
+ find /var/log/wtmp -type f -user root -group root -perm 640
+ CHECK_RESULT 0 0 0 'check /var/log/wtmp file right fail'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='check /var/log/wtmp file right fail'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ LOG_INFO 'Finish testcase execution.'
+ message='Finish testcase execution.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Finish testcase execution.'
Sat Apr 29 02:44:52 2023 - INFO  - Finish testcase execution.
+ CASE_RESULT 0
+ case_re=0
+ test -z 2
+ test 2 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Sat Apr 29 02:44:53 2023 - ERROR - failed to execute the case.
+ exit 2
+ POST_TEST_DEFAULT
+ LOG_INFO 'oe_test_check_file_sys_protect_003.sh post_test'
+ message='oe_test_check_file_sys_protect_003.sh post_test'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'oe_test_check_file_sys_protect_003.sh post_test'
Sat Apr 29 02:44:53 2023 - INFO  - oe_test_check_file_sys_protect_003.sh post_test
